lib/ansible/playbook/base.py

```diff
@@ -112,6 +112,9 @@ def _generic_g(prop_name, self):
         if hasattr(self, method):
             return getattr(self, method)()
 
+        if prop_name not in self._attributes:
+            return None
+
```

Comment:
@mattclay ^  It seemed like you understood this thoroughly -- want to let me know what the comment should say?